### PR TITLE
Base functionality to create movies from images

### LINKED ISSUE #469: Function to create movies
The DQ Office creates many movies from static plots. Another researcher is working on a project that would benefit from a simple way to create moves from plots. We should create a function to make movies from a list of images. Finding a mostly Python way to do this would be best.

### PR DESCRIPTION
**Pull request by AdamTheisen**
- [x] Closes #469 
- [x] Tests added
- [x] Documentation reflects changes
- [x] PEP8 Standards or use of linter
- [x] Xarray Dataset or DataArray variable naming follows 'ds' or 'da' naming

### CODE REVIEW

File: act/tests/utils/test_io_utils.py [hunk 1]
```diff
@@ -203,3 +204,11 @@ def test_gunzip():
             assert file.endswith('.nc')
 
         assert Path(unpack_filename).is_file() is False
+
+
+def test_generate_movie():
```
Comment (zssherman) [inline act/tests/utils/test_io_utils.py:209]: I would make the test optional as well

File: act/utils/io_utils.py [hunk 2]
```diff
@@ -276,3 +282,55 @@ def unpack_gzip(filename, write_directory=None, remove=False):
...
+
+    Returns
+    -------
+    write_filename : str
+        Full path name of created gunzip file
+
+    """
+    if not MOVIEPY_AVAILABLE:
+        raise ImportError(
+            'Cartopy needs to be installed on your ' 'system to make geographic display plots.'
```
Comment (zssherman) [inline act/utils/io_utils.py:317]: Cartopy to Moviepy

**Review by zssherman — CHANGES_REQUESTED** [review]

**Comment by AdamTheisen** [comment]
I had a feeling that the test was not going to work on Windows... I'll try and test that out some more when I get a chance.

**Comment by AdamTheisen** [comment]
@zssherman this should be good to go. 

@kenkehoe we can test this out as an optional dependency for now and can always adjust in the future.  You can pass it a list of images or a list of URLs and is pretty straight forward at the moment.

**Review by zssherman — APPROVED** [review]

**Pull request merged**